localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
 
- prince_nightborne_male:0 "Prince"
- princess_nightborne_female:0 "Princess"
- principality_nightborne:0 "Principality"
+ ### Zandalari ###
+ # Feudal
+ god_king_feudal:0 "God King"
+ god_queen_feudal:0 "God Queen"
 
- lord_nightborne_male:0 "Lord"
- lady_nightborne_female:0 "Lady"
+ prince_male_zandalari_kingdom:0 "Prince"
+ princess_female_zandalari_kingdom:0 "Princess"
+ prince_male_zandalari_empire:0 "Prince"
+ princess_female_zandalari_empire:0 "Princess"
 
+ ### Blood trolls ###
+ # Feudal
  blood_troll_male:0 "#"
- blood_troll_count_female:0 "Warmother"
- blood_troll_duke_female:0 "Grand Ma'da"
- blood_troll_queen_female:0 "Queen Ma'da"
- blood_troll_empress_female:0 "Divine Ma'da" lady_nightborne_female:0 "Lady"
+ warmother_female_blood_troll:0 "Warmother"
+ grand_mada_female_blood_troll:0 "Grand Ma'da"
+ queen_mada_female_blood_troll:0 "Queen Ma'da"
+ divine_mada_female_blood_troll:0 "Divine Ma'da" 
  
- lich_king_male:0 "Lich King"
- lich_king_female:0 "Lich Queen"
+ ### Drakkari ###
+ # Feudal
+ frost_king_feudal:0 "Frost King"
+ frost_queen_feudal:0 "Frost Queen"
+
+ ### Dragons ###
+ dragonlord_male_dragon:0 "Dragon Lord"
+ dragonlady_female_dragon:0 "Dragon Lady"
+ lord_male_dragon:0 "Lord"
+ lady_female_dragon:0 "Lady"
+ overlord_male_dragon:0 "Overlord"
+ overlady_female_dragon:0 "Overlady"
+
+ prince_male_dragon_kingdom:0 "Prince"
+ princess_female_dragon_kingdom:0 "Princess"
+ prince_male_dragon_empire:0 "Prince"
+ princess_female_dragon_empire:0 "Princess"
+
+ ### Red Dragons ###
+ dragonking_male_red_dragon:0 "Dragon King"
+ dragonqueen_female_red_dragon:0 "Dragon Queen"
+ dragonprince_male_red_dragon:0 "Dragon Prince"
+ dragonprincess_female_red_dragon:0 "Dragon Princess"
+
+ ### Mantids ###
+ emperor_male_mantid:0 "Grand Emperor"
+ empress_female_mantid:0 "Grand Empress"
+
+ imperial_vizier_male_mantid:0 "Imperial Vizier"
+ imperial_vizier_female_mantid:0 "Imperial Vizier"
+ grand_vizier_male_mantid:0 "Grand Vizier"
+ grand_vizier_female_mantid:0 "Grand Vizier"
+ vizier_male_mantid:0 "Vizier"
+ vizier_female_mantid:0 "Vizier"
+
+ ### Qiraji ###
+ # Special
+ emperor_ahn_qiraj_male_qiraji:0 "Emperor"
+ empress_ahn_qiraj_female_qiraji:0 "Emperor"
+ empire_ahn_qiraj:0 "Empire"
+
+ # Others
+ lord_male_qiraji:0 "Lord"
+ lady_female_qiraji:0 "Lady"
+ hive_qiraji:0 "Hive"
+ high_overlord_male_qiraji:0 "High Overlord"
+ high_overlady_female_qiraji:0 "High Overlady"
+ brood_qiraji:0 "Brood"
+
+ ### Wildhammer ###
+ # Feudal
+ chief_male_wildhammer:0 "Chief"
+ chieftess_female_wildhammer:0 "Chieftess"
+ chiefdom_wildhammer:0 "Chiefdom"
+ thane_male_wildhammer:0 "Thane"
+ thane_female_wildhammer:0 "Thane"
+ high_chiefdom_wildhammer:0 "High Chiefdom"
+ high_thane_male_wildhammer:0 "High Thane"
+ high_thane_female_wildhammer:0 "High Thane"
+
+ ### Dark Iron ###
+ # Feudal
+ lord_male_darkiron:0 "Lord"
+ lady_female_darkiron:1 "Lady"
+ lordship_darkiron:0 "Lordship"
+ thane_male_darkiron:0 "Thane"
```

Comment:
Should be emperor even though it's kingdom title (https://wowpedia.fandom.com/wiki/Dagran_Thaurissan).